src/Form/FormAnalyzer.js

```diff
@@ -229,6 +229,15 @@ class FormAnalyzer {
         });
     }
 
+    evaluatePasswordHints() {
+        if (this.form.textContent) {
+            const hasPasswordHints = safeRegexTest(this.matching.getDDGMatcherRegex('passwordHintsRegex'), this.form.textContent, 200);
+            if (hasPasswordHints) {
+                this.increaseSignalBy(5, 'Password hints');
```

Comment:
I have given a much stronger score to hints, as they seem to be a very clear indicator.